src/faebryk/library/NetTie.py

```diff
@@ -0,0 +1,51 @@
+# This file is part of the faebryk project
+# SPDX-License-Identifier: MIT
+
+import logging
+from enum import Enum
+
+import faebryk.library._F as F
+from faebryk.core.module import Module
+from faebryk.core.moduleinterface import ModuleInterface
+from faebryk.library.Electrical import Electrical
+from faebryk.library.KicadFootprint import KicadFootprint
```

Comment:
Avoid direct imports from faebryk.library, use F.

Reply:
I'm pulling this into my project for now and separating the PRs.
Here's another draft for it specifically: https://github.com/atopile/faebryk/pull/39